src/pages/GovPages/Proposals/ProposalCard.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useContext, useState } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import { Clock } from 'react-feather'
+import { Flex } from 'rebass'
+
+import { TYPE } from '../../../theme'
+import Row, { RowBetween } from '../../../components/Row'
+import { AutoColumn } from '../../../components/Column'
+import Card, { LightCard } from '../../../components/Card'
+
+import useInterval from '../../../hooks/useInterval'
+import HourGlass from '../../../assets/svg/hourglass.svg'
+import Block from '../../../assets/svg/block.svg'
+import Done from '../../../assets/svg/done_all.svg'
+
+const Container = styled(LightCard)<{ isPassed: number }>`
+  // 0 - voting in progress
+  // 1 - passed
+  // 2 - failed
+  padding: 1.25rem;
+  height: 88px;
+  position: relative;
+  cursor: pointer;
+  background: ${({ isPassed, theme }) =>
+    isPassed === 0
+      ? `linear-gradient(113.18deg, rgba(255, 255, 255, 0.35) -0.1%, rgba(0, 0, 0, 0) 98.9%), ${theme.dark1}`
+      : `linear-gradient(141.72deg, ${theme.bg1} -11.46%, ${
+          isPassed === 1 ? 'rgba(5, 122, 85, 0.5)' : 'rgba(238, 46, 81, 0.5)'
+        } 180.17%)`};
+  background-blend-mode: overlay, normal;
+  border: none;
+
+  ::before {
```

Comment:
Could you please explain this one to me?

Reply:
This is for the border of proposal card.